<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>411</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>选择答案：</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>391</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="radioButtonA">
       <property name="text">
        <string>RadioButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonB">
       <property name="text">
        <string>RadioButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonC">
       <property name="text">
        <string>RadioButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonD">
       <property name="text">
        <string>RadioButton</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>421</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>170</y>
     <width>151</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>耗时(s)&amp;&amp;得分</string>
   </property>
   <widget class="QLabel" name="labelRunTime">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>120</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelScore">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>120</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonCommit">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>620</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNext">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>620</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>41</y>
     <width>641</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_1">
      <property name="baseSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>26</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="baseSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>26</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="baseSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>26</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>3</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="baseSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>26</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>4</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="baseSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>26</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>5</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="baseSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>26</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>6</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="baseSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>26</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>7</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="baseSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>26</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>8</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="baseSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>26</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>9</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="baseSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>26</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>10</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
